index.html

```diff
@@ -11,51 +11,187 @@
+                don’t hesitate to contact me.
+                </p>
+            
+
+                <ul class="page-links">
+                    <li>
+                        <a class="icon-list" href="https://github.com/iLynette"><i class="fab fa-github"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://www.linkedin.com/in/lynette-acholah/"><i class="fab fa-linkedin"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://github.com/iLynette"><i class="fab fa-angellist"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://twitter.com/acholah_lynette"><i class="fab fa-twitter"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://medium.com/@acholahlynette111"><i class="fab fa-medium"></i></a>
+                    </li>
+                </ul>
+            </div>
+
+
+        </section>
+
+        <section class="my-projects">
+            <div class="down-arrow">
+                <img src="./images/arrow.png" alt="down-arrow">
+            </div>
+
+            <div class="title">
+                <h2>My Recent Works</h2>
+            </div>
+
+            <div class="line">
+                <img src="./images/greenline.png" alt="greenline">
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
```

Comment:
- [ ] It would be great to apply the button interactions, as shown above, 👍 for improving the User Interface.